ChangeLog

```diff
@@ -102,6 +102,10 @@ What's New in Pylint 1.8?
 
       Close #574
 
+    * Disabling ``function-redefined```message for function or method with
```

Comment:
Mark-up is mixed up (two opening \`, three closing \`)
Maybe we can come up with more expressive description of change? It shouldn't mention disabling, as it'll introduce confusion with `pylint: disable=...` comments.

Maybe something along the lines of:

``function-redefined`` is no longer emitted for functions and methods which name matches dummy variable name regular expression.

Reply:
Ok. Thanks @rogalski .